src/OpenTelemetry.Contrib.Instrumentation.Owin/OwinInstrumentationOptions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// <copyright file="OwinInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using Microsoft.Owin;
+using OpenTelemetry.Context.Propagation;
+
+namespace OpenTelemetry.Contrib.Instrumentation.Owin
+{
+    /// <summary>
+    /// Options for requests instrumentation.
+    /// </summary>
+    public class OwinInstrumentationOptions
+    {
+        /// <summary>
+        /// Gets or sets <see cref="TextMapPropagator"/> for context propagation. Default value: <see cref="Propagators.DefaultTextMapPropagator"/>.
+        /// </summary>
+        public TextMapPropagator Propagator { get; set; } = Propagators.DefaultTextMapPropagator;
```

Comment:
Why a separate `Propagator` option specifically for Owin? Can't the middleware just use `Propagators.DefaultTextMapPropagator` directly and then if people want to change it they can use `Sdk.SetDefaultTextMapPropagator`?

Reply:
TBH, I copied this from WCF and I don't remember why it was done there 🤣 You want me to remove?

Reply:
we *used* to have per instrumentation propagator (in asp.net core, httpclient etc). But then it got removed in favor of the Propagators.Default. Probably a leftover from old days..
We can remove it and be consistent.

Reply:
Updated